src/main/java/ladder/domain/Line.java

```diff
@@ -0,0 +1,80 @@
+package ladder.domain;
+
+import ladder.domain.exception.DifferentLineSizeException;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Line {
+
+    private final List<Stick> sticks;
+
+    public Line(final List<Stick> sticks) {
+        this.sticks = sticks;
+    }
+
+    public static Line from(final List<Boolean> values) {
+        List<Stick> sticks = values.stream().map(Stick::new).collect(Collectors.toList());
```

Comment:
객체지향 생활 체조 원칙의 `규칙 4: 한 줄에 점을 하나만 찍는다.`를 적용해보면 어떨까요?
```suggestion
        List<Stick> sticks = values.stream()
            .map(Stick::new)
            .collect(Collectors.toList());
```

Reply:
앗 기본적인건데 이 부분에서 잊고 었던것 같군요ㅠ